pkg/infra/log/file_test.go

```diff
@@ -17,30 +19,27 @@ func (w *FileLogWriter) WriteLine(line string) error {
 }
 
 func TestLogFile(t *testing.T) {
-	Convey("When logging to file", t, func() {
-		fileLogWrite := NewFileWriter()
-		So(fileLogWrite, ShouldNotBeNil)
-
-		fileLogWrite.Filename = "grafana_test.log"
-		err := fileLogWrite.Init()
-		So(err, ShouldBeNil)
-
-		Convey("Log file is empty", func() {
-			So(fileLogWrite.maxlines_curlines, ShouldEqual, 0)
-		})
-
-		Convey("Logging should add lines", func() {
-			err := fileLogWrite.WriteLine("test1\n")
-			So(err, ShouldBeNil)
-			err = fileLogWrite.WriteLine("test2\n")
-			So(err, ShouldBeNil)
-			err = fileLogWrite.WriteLine("test3\n")
-			So(err, ShouldBeNil)
-			So(fileLogWrite.maxlines_curlines, ShouldEqual, 3)
-		})
-
-		fileLogWrite.Close()
-		err = os.Remove(fileLogWrite.Filename)
-		So(err, ShouldBeNil)
+	fileLogWrite := NewFileWriter()
+	require.NotNil(t, fileLogWrite)
+
+	fileLogWrite.Filename = "grafana_test.log"
+	err := fileLogWrite.Init()
+	require.NoError(t, err)
+
+	assert.Zero(t, fileLogWrite.maxlines_curlines)
+
+	t.Run("adding lines", func(t *testing.T) {
+		err := fileLogWrite.WriteLine("test1\n")
+		require.NoError(t, err)
+		err = fileLogWrite.WriteLine("test2\n")
+		require.NoError(t, err)
+		err = fileLogWrite.WriteLine("test3\n")
+		require.NoError(t, err)
+
+		assert.Equal(t, 3, fileLogWrite.maxlines_curlines)
 	})
+
+	fileLogWrite.Close()
+	err = os.Remove(fileLogWrite.Filename)
+	require.NoError(t, err)
```

Comment:
Should these be put in a cleanup call, to be sure the file gets removed?

Reply:
Nice catch, I didn't notice that `os.Remove` 🙈